doc/adrs/070_contribute_a_diagram_node_style.adoc

```diff
@@ -0,0 +1,90 @@
+= ADR-070 - Contribute a diagram node style 
+
+== Context
+
+Currently, the diagram styles are hardcoded and known by the sirius-component core in the backend and in the frontend.
+
+The aim of this adr is to explain how to contribute a new diagram style from an additional project and what changes are needed in the sirius-component projects to support the contributed style.
+
+=== Current state
+On the backend, style are managed specifically for
+* the rendering in  `sirius-components-diagrams`
+* the layout in `sirius-components-diagrams-layout`
+** ELK layout
+** Incremental layout
+* the service called by the front end
+
+On the frontend, styles are managed with a dedicated sprotty inherited view.
+
+== Enhancement ==
+
+For the rest of this adr, let's consider that the contributed style is a parametric svg style node.
```

Comment:
It's the first time this term is used ("parametric svg node style") and no explanation is given about what it means or what it's usfule for.